android/app/src/main/cpp/audio_mixer.cpp

```
@@ -48,14 +49,86 @@ void AudioMixer::updateDeviceAudio(int deviceId, const int16_t* audioData, int n
+    {
+        std::lock_guard<std::mutex> lock(deviceRegistryMutex);
+        auto it = devices.find(deviceId);
+        if (it != devices.end()) {
+            device = it->second;
+        }
+    }
+    if (!device) {
+        return;
+    }
+
+    const uint32_t prevSeq = device->lastSeq;
```

Clang (14.0.6)

[warning] variable 'prevSeq' is not initialized

(cppcoreguidelines-init-variables)